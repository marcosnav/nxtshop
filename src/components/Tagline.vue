<template>
  <h2 class="Tagline">
    <slot />
  </h2>
</template>

<script lang="ts">
import Vue from 'vue';
export default Vue.extend({});
</script>

<style lang="sass" scoped>
.Tagline
  color: var(--sub-text)
  font-size: 0.7em
  font-weight: 400
  margin: 0
  position: relative
</style>
