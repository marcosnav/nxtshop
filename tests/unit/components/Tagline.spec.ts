import { shallowMount } from '@vue/test-utils';
import Tagline from '@/components/Tagline.vue';

describe('<Tagline>', () => {
  const component = shallowMount(Tagline, {
    slots: {
      default: 'Smart tech wrapping your wrist',
    },
  });

  test('renders properly', () => {
    expect(component.classes('Tagline')).toBeTruthy();
    expect(component.text()).toMatch('Smart tech wrapping your wrist');
    expect(component.element).toMatchSnapshot();
  });
});
