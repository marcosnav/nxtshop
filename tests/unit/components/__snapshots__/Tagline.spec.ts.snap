// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tagline> renders properly 1`] = `
<h2
  class="Tagline"
>
  Smart tech wrapping your wrist
</h2>
`;
